Cargo.toml

```diff
@@ -27,18 +28,19 @@ fs_extra = "1"
 futures = "0.3"
 http-types = "2"
 indicatif = "0.15"
-nipper = "0.1"
+nipper = { git = "https://github.com/thedodd/nipper.git", branch = "iter-attrs" }
 notify = "4"
 open = "1"
+remove_dir_all = "0.6"
 sass-rs = "0.2.2"
 seahash = "4"
-serde = { version="1", features=["derive"] }
+serde = { version = "1", features = ["derive"] }
 structopt = "0.3"
 structopt-derive = "0.4"
 surf = "2"
-tide = { version="0.16.0", features=["unstable"] }
+tide = { version = "0.16.0", features = ["unstable"] }
+tide-websockets = { git = "https://github.com/http-rs/tide-websockets.git", rev = "270f408cdf4e5ee2bd28c7f5fcb57e5085d49ead" }
```

Comment:
@jbr thanks for your work on this. I'll update to the next release (which supports sending raw messages) once it lands. It works quite nicely!